src/CommitmentsV2/SFA.DAS.CommitmentsV2/Services/AcademicYearDateProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+
+namespace SFA.DAS.CommitmentsV2.Services
+{
+    public class AcademicYearDateProvider : IAcademicYearDateProvider
+    {
+        private readonly ICurrentDateTime _currentDateTime;
+
+        public AcademicYearDateProvider(ICurrentDateTime currentDateTime)
+        {
+            _currentDateTime = currentDateTime;
+        }
+
+        public DateTime CurrentAcademicYearStartDate
+        {
+            get
+            {
+                var now = _currentDateTime.Now;
+                var cutoff = new DateTime(now.Year, 8, 1);
```

Comment:
Does the date need to be specified as UTC? To avoid comparing local to UTC date times I mean.
DateTime.SpecifyKind(now, DateTimeKind.Utc);